lib/toolshed/unix.ex

```diff
@@ -65,38 +65,47 @@ defmodule Toolshed.Unix do
   end
 
   @doc """
-  Print out the date similar to the Unix date command
+  Print out the date similar to the Unix date command when the envoriment variable "LC_ALL=C" is exported
```

Comment:
```suggestion
  Return the date and time in UTC
```

I rephrased this since the important part is that it returns the date and time. I would expect people familiar with the Unix `date` command to recognize the similarity and for people unfamiliar with it to just think of it as a convenience helper function. The details of how it works aren't important and possibly in the future, we'll support other timezones and make it locale aware.